src/datasets/tasks/__init__.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+from .base import TaskTemplate
+from .question_answering import QuestionAnswering
+from .text_classification import TextClassification
+
+
+__all__ = ["TaskTemplate", "QuestionAnswering", "TextClassification"]
+
+
+NAME2TEMPLATE = {QuestionAnswering.task: QuestionAnswering, TextClassification.task: TextClassification}
+
+
+def task_template_from_dict(task_template_dict: dict) -> Optional[TaskTemplate]:
```

Comment:
Is this function supposed to be used by the user? If so it should have a doc string and doc

Reply:
no this is not supposed to be used by the user (at least for now). still no harm in having a docstring, so i've added:

>     """Create one of the supported task templates in :obj:`datasets.tasks` from a dictionary."""